### ISSUE TITLE
[Section] Notify the user for ledger & browser compatibility

### ISSUE THREAD

**Issue by goga-m** [PayvoHQ/wallet#929]
### Describe the refactor you'd like to request

Need to adjust UI and notify the user whether the ledger connection is supported by the browser.

See https://caniuse.com/webhid and https://caniuse.com/webusb

This affects the Ledger Wallet Import flow as well as the Transaction Flow using a ledger wallet.

**Comment by alfonsobries** [comment]
@goga-m wouldn't it be better to disable import ledger wallets when the ledger is not compatible?

Something like this 
![image](https://user-images.githubusercontent.com/17262776/152585290-6679244a-10a8-4633-99ed-8523036cddaf.png)

For the rest of the transactions as far as I can tell the only way that the users have ledger wallets in the browsers is if they import profile exported in another browser where they were able to import them at first, we can restrict those wallets with  a message like "your wallet xxxx requires ledger connection which is not compatible with this browser" or we can simply disable the transactions with the same tooltip

wdyt?

**Comment by goga-m** [comment]
Yeah basically thinking that the only restrictions could be only when signing, importing wallets or sending transactions so that the user can always view already existing ledger wallets,  monitor transactions, votes and other info.
